partitura/io/importmatch.py

```diff
@@ -591,7 +591,9 @@ def part_from_matchfile(
         if "accent" in note.ScoreAttributesList:
             articulations.add("accent")
         if "leftOutTied" in note.ScoreAttributesList:
-            continue
+            # continue introduces inconsistency when save_match
+            # continue
+            pass
```

Comment:
Should we keep this open for a less hacky fix?

Reply:
Yes definitely.

Reply:
This needs a proper fix, even after the PR is merged.